### PR TITLE
Support multilang quiz and course names.

### PR DESCRIPTION
**Pull request by lucaboesch**

### CODE REVIEW

File: classes/quiz.php [hunk 1]
```diff
@@ -258,7 +258,7 @@ public function get_quiz_data(int $quizid): \stdClass {
         // Dashboard link.
         $dashboardlink = new \moodle_url('/local/assessfreq/dashboard_quiz.php', array('id' => $quizid));
 
-        $quizdata->name = $quizrecord->name;
+        $quizdata->name = format_string($quizrecord->name, true, ["context" => $context, "escape" => true]);
```
Comment (dmitriim) [inline classes/quiz.php:261]: Would you like to apply format_string to $course->fullname? 

https://github.com/catalyst/moodle-local_assessfreq/pull/107/files#diff-f11bab2e9c4891c6f8be7489d8f12510c5e88b5a676e76a158072980df88d549R277

**Comment by dmitriim** [comment]
Hi @lucaboesch! Thank you for your patch!

It looks good in general. Would be good to rebase on top of the latest master if possible. Also, have you considered to add some test coverage to those changes?

File: classes/quiz.php (hunk 1 above)
Comment (lucaboesch) [reply classes/quiz.php:261]: Done, @dmitriim !
Good point, thanks.

**Comment by lucaboesch** [comment]
Pull request updated.

**Comment by dmitriim** [comment]
Looks like CI issues are not related to the updated code. Merging it in...

**Pull request merged**